lib/utapi/utilities.js

```diff
@@ -6,7 +6,12 @@ const { UtapiClient, utapiVersion } = require('utapi');
 const logger = require('../utilities/logger');
 const _config = require('../Config').config;
 // setup utapi client
-const utapi = new UtapiClient(_config.utapi);
+const enableClient = utapiVersion === 2 || !!_config.utapi;
+const utapi = new UtapiClient({
+    ..._config.utapi,
+    tls: _config.certFilePaths,
+    enableClient
```

Comment:
looking at https://github.com/scality/utapi/pull/864/files#diff-f9226a8a10ea61b9c4e485d5419770d9001d76c89c884f5ef3d2fd776e11f346R119-R127

should this be `disableClient`?